pom.xml

```diff
@@ -3100,6 +3100,15 @@
                 <mutator>TRUE_RETURNS</mutator>
                 <mutator>VOID_METHOD_CALLS</mutator>
               </mutators>
+              <avoidCallsTo>
+                <!-- Till https://github.com/hcoles/pitest/issues/766
+                 Pitest complains about redundant call to close() method.
+                 This is unavoidable, since this method is called deep inside
+                 try-with-resources statement. As a workaround, we can exclude
+                 all methods of these classes from mutation. -->
+                <avoidCallsTo>java.io.BufferedReader</avoidCallsTo>
+                <avoidCallsTo>java.io.Reader</avoidCallsTo>
```

Comment:
Why exclude all methods of the classes and not just the specific methods that are in issue?

Reply:
I have no idea how to exclude a method. I've tried
```xml
                <avoidCallsTo>java.io.Reader.close</avoidCallsTo>
                <avoidCallsTo>java.io.Reader#close</avoidCallsTo>
                <avoidCallsTo>java.io.Reader#close()</avoidCallsTo>
```
None of them works.  There is an [issue](https://github.com/hcoles/pitest/issues/404) for this in the pitest repo.

Reply:
Ok, then lets ignore.